### DISCUSSION TITLE
Issue with Alt Tab and Win Tab on latest version?

### DISCUSSION THREAD

**Discussion by tyse-2001** [valinet/ExplorerPatcher#2016] [General]
Recently after installing the latest version, my PC had issues with loading Task View and switching applications via Alt Tab. I troubleshooted this for a while unsuccessfully before uninstalling it to see if it was an issue with ExplorerPatcher issue and apparently, that did the trick. Redownloading caused the issue to reappear. Is anyone else facing something similar and if so are there any workarounds?

**Comment by Poopooracoocoo** [comment]
See #1687
